File: TeeNew/Builder2006/Gallery_Alternate.cpp
//---------------------------------------------------------------------------

#include <vcl.h>
#pragma hdrstop

#include "Gallery_Alternate.h"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma resource "*.dfm"
TGalleryAlternate *GalleryAlternate;
//---------------------------------------------------------------------------
__fastcall TGalleryAlternate::TGalleryAlternate(TComponent* Owner)
  : TForm(Owner)
{
}
//---------------------------------------------------------------------------
void __fastcall TGalleryAlternate::FormShow(TObject *Sender)
{
  Gallery = new TTeeGalleryForm(this);
  Gallery->Align = alClient;
  AddFormTo(Gallery,this,0);
}
//---------------------------------------------------------------------------
void __fastcall TGalleryAlternate::FormDestroy(TObject *Sender)
{
  delete Gallery;  
}
//---------------------------------------------------------------------------

File: TeeNew/Gallery_Alternate.pas
unit Gallery_Alternate;
{$I TeeDefs.inc}

interface

uses
  {$IFNDEF LINUX}
  Windows, Messages,
  {$ENDIF}
  SysUtils, Classes,
  {$IFDEF CLX}
  QGraphics, QControls, QForms, QDialogs, QExtCtrls, QStdCtrls, QComCtrls,
  {$ELSE}
  Graphics, Controls, Forms, Dialogs, ExtCtrls, StdCtrls, ComCtrls,
  {$ENDIF}
  TeeGalleryAlternate, TeePenDlg;

type
  TGalleryAlternate = class(TForm)
    procedure FormShow(Sender: TObject);
    procedure FormDestroy(Sender: TObject);
  private
    { Private declarations }
    Gallery : TTeeGalleryForm;
  public
    { Public declarations }
  end;

implementation

{$IFNDEF CLX}
{$R *.DFM}
{$ELSE}
{$R *.xfm}
{$ENDIF}

procedure TGalleryAlternate.FormShow(Sender: TObject);
begin
  Gallery:=TTeeGalleryForm.Create(Self);
  Gallery.Align:=alClient;
  AddFormTo(Gallery,Self);
end;

procedure TGalleryAlternate.FormDestroy(Sender: TObject);
begin
  Gallery.Free;
end;

initialization
  RegisterClass(TGalleryAlternate);
end.
